mlir/include/mlir/Dialect/Rock/IR/RockOps.td

```diff
@@ -50,6 +50,12 @@ class TensorOrMemRefRankOf<list<Type> allowedTypes, list<int> ranks> :
              /*summary=*/"",
              "::mlir::ShapedType">;
 
+class TensorOrMemRefOf<list<Type> allowedTypes> :
```

Comment:
... hold on, didn't we already have this? ... Ok, since you added it, I guess not